pkg/proxy/benchmarking/.gitignore

```diff
@@ -0,0 +1,2 @@
+# golang benchmarking results
+*.report
```

Comment:
We usually write things under `_output` and have helpers for this. see `hack/lib`

Reply:
got it